src/css/finesse.css

```diff
@@ -773,3 +773,49 @@ picture {
   border: 1px solid var(--border-color);
   border-radius: var(--spacing-4);
 }
+
+/* Modal */
+
+.modal-overlay {
+  display: none;
+  position: fixed;
+  inset: 0;
+  background-color: rgb(0, 0, 0, 0.5);
+  transition: 0.3s;
+  z-index: 10;
+  inline-size: 100vw;
+  block-size: 100vh;
+  user-select: none;
+}
+
+.modal-container {
+  background-color: var(--bg-color-light);
+  inline-size: clamp(18rem, 40vw, 30rem);
```

Comment:
Why are you using vw unit with rem?

Reply:
This is so that the modal width is responsive in mobile devices also. and the modal width increases/decreases fluidly when resizing the browser window.